tools/ramclustr/ramclustr_define_experiment.xml

```diff
@@ -1,8 +1,12 @@
-<tool id="ramclustr_define_experiment" name="RAMClustR define experiment" version="1.0.2" profile="21.09">
+<tool id="ramclustr_define_experiment" name="RAMClustR define experiment" version="1.0.2+galaxy0" profile="21.09">
```

Comment:
I think there is no reason to introduce this extra versioning layer since the tool itself exists only here. I would just bump the version to `1.0.3` or so.

Reply:
I think it makes sense to separate the version of the python code from the Galaxy wrapper (for what it is worth?)